core/lib/zksync_core/src/state_keeper/io/event_indexes_migration.rs

```diff
@@ -0,0 +1,378 @@
+//! Temporary module for migrating fee addresses from L1 batches to miniblocks.
+
+use std::time::Duration;
+
+use anyhow::Context as _;
+use tokio::sync::watch;
+use zksync_dal::{ConnectionPool, StorageProcessor};
+use zksync_types::MiniblockNumber;
+
+/// Runs the migration for non-pending miniblocks. Should be run as a background task.
```

Comment:
Nit: In the context of the fee address migration, "non-pending miniblocks" means miniblocks belonging to a sealed L1 batch. This distinction is made because the fee address is copied to miniblocks from the corresponding L1 batch record, which is not present in Postgres for pending miniblocks. AFAIU, this distinction is not applicable for this migration.

Reply:
Yes, this one was mostly copy-pasted from fee address migration PR and then adjusted, but in the context of these changes, new events are saved already correctly. So that is just comment being not updated